### PR TITLE
Update and test dbinit

### PR DESCRIPTION
**Pull request by gsilvis**
Supersedes #640

### CODE REVIEW

**Comment by gsilvis** [comment]
@henn, please review

**Comment by knikolla** [comment]
@henn I don't agree having this as part of the CI.

If you want CI to test this, write a integration test that tests it and its correct output. Just running it is not enough.

**Comment by henn** [comment]
Kristi - my goal with including this in CI was to ensure that a future
change to the API wouldn't break the example script accidentally; the
script isn't intended to be part of our testing per se.

Is your opinion on this strong? How do others feel?

File: scripts/haas [hunk 1]
```diff
@@ -14,5 +14,8 @@
 # express or implied.  See the License for the specific language
 # governing permissions and limitations under the License.
 
+import sys
+
 from haas import cli
-cli.main()
+
+sys.exit(cli.main())
```
Comment (henn) [inline scripts/haas]: cli.main() doesn't return a value currently.

I was thinking that maybe cli.main() should do try/except with the proper exceptions, then call sys.exit() itself.

What do you think?

Comment (gsilvis) [inline scripts/haas]: Good point.  Right now, ``cli.main`` calls ``sys.exit`` manually on errors, but not on normal results.  So we can just revert this part of the change.  (I copied it over from your PR without thinking about it very hard.)

**Comment by shwsun** [comment]
@knikolla Is your concern addressed?

**Comment by gsilvis** [comment]
It would be good to actually test that the script does the right thing, but just testing that it runs at all is a start.  (Note that we had to fix it here because previous changes to HIL made it invalid---adding it to CI in this form will prevent that from happening again.)

**Pull request merged**